client/my-sites/checkout/composite-checkout/test/composite-checkout-thank-you.js

```diff
@@ -749,7 +722,6 @@ describe( 'getThankYouPageUrl', () => {
 	} );
 
 	it( 'redirects to business upgrade nudge if concierge and jetpack are not in the cart, and premium is in the cart', () => {
```

Comment:
Shouldn't we also update the test descriptions to no longer mention the Concierge product?

```suggestion
	it( 'redirects to business upgrade nudge if jetpack is not in the cart, and premium is in the cart', () => {
```

Reply:
Done in 6ab4db7. Good catch on this one!